lms/static/js/demographics_collection/DemographicsCollectionBanner.jsx

```diff
@@ -8,47 +9,78 @@ export class DemographicsCollectionBanner extends React.Component {
     super(props);
     this.state = {
       modalOpen: false,
+      hideBanner: false
+    }
+
+    this.dismissBanner = this.dismissBanner.bind(this);
+  }
+
+  async dismissBanner(e) {
+    // The dismiss button is nested inside an <a>, so we need to call stopPropogation here to
+    // keep the Modal from also opening when the Dismiss button is clicked
+    e.stopPropagation();
+
+    const requestOptions = {
+      method: 'PATCH',
+      credentials: 'include',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRFTOKEN': Cookies.get('csrftoken'),
+      },
+      body: JSON.stringify({
+        show_call_to_action: false,
+      })
+    };
+
+    let response = {};
+    response = await fetch(`${this.props.lmsRootUrl}/api/demographics/v1/demographics/status/`, requestOptions);
+    if (response.status === 200) {
```

Comment:
Shouldn't we just always hide the banner? even if the patch doesn't stick? I would think it would feel weird to hit close and not have it disapear. @staubina Any thoughts?

Reply:
Ah, I see what you're saying and I'm onboard.

Reply:
Yes